### ISSUE TITLE
travis: ElasticSearch 2.2.0

### ISSUE THREAD

**Issue by jirikuncar** [inveniosoftware/invenio-search#58]
I'm testing installation process for ElasticSearch 2.2.0 on Travis CI (current 1.4.0) for Invenio-OAIServer.

```diff
-services:
-  - elasticsearch

env:
+  - ES_VERSION=2.2.0

before_install:
+  - "mkdir /tmp/elasticsearch"
+  - "wget -O - https://download.elasticsearch.org/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/${ES_VERSION}/elasticsearch-${ES_VERSION}.tar.gz | tar xz --directory=/tmp/elasticsearch --strip-components=1"
+  - "/tmp/elasticsearch/bin/elasticsearch -d -D es.path.data=/tmp -D es.index.store.type=memory -D es.discovery.zen.ping.multicast.enabled=false"
```

### Proposal

- lowest: ES_VERSION=~~2.0.1~~ 2.2.0
- released: ES_VERSION=~~2.1.0~~ 5.0.0/latest?
- dev: ES_VERSION=~~2.2.0~~ dev
(updated: 2016-10-24)

--
see https://github.com/jirikuncar/invenio-oaiserver/commit/b2041e33f0ca5b3236ae000713acc33c556e9704

**Comment by jirikuncar** [comment]
We should have a look at supporting latest Elasticsearch 5.0 and unpin `elasticsearch` dependency created in #87.

**Comment by jirikuncar** [comment]
closed via ecd50fb53ed89d755952fafc823e31e04a1da4a2
